manifests/a/Argotronic/ArgusMonitor/6.2.1.2620/Argotronic.ArgusMonitor.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Automatically updated by the winget bot at 2022/Dec/22
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.1.2620
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 6.1.3.2605

```

Comment:
```suggestion
- DisplayVersion: 6.2.1.2620
```

Reply:
The Apps & Features entry seems to match the package version now. 
![image](https://user-images.githubusercontent.com/114952565/209235093-c7649711-bd94-4aff-a1b5-fa5c636a014c.png)

Reply:
@stephengillie - The publisher markets the version as `6.2.01` on their official website https://www.argusmonitor.com/, so to avoid ambiguity for the end user using `winget install`, the PackageVersion field is set to Marketing version `6.2.01` and for exact matching with Apps&Features, ARP Entries are used in the manifest.